### PR TITLE
Create security group for load balancer

### PR DESCRIPTION
**Pull request by kenoir**
This PR creates a security group for the load balancer, rather than rely on one existing.

### CODE REVIEW

**Comment by philwills** [comment]
Hmm, that certainly used to be a 'magic' AWS name that just worked. My guess would be that this was a VPC thing, but I thought I had in the changes that @rtyley made so that membership could work on VPC.

**Comment by rtyley** [comment]
I never got a far as actually pull-requesting the changes, so unfortunately they were never merged! 

https://github.com/guardian/status-app/tree/vpc-cloud-formation

**Comment by philwills** [comment]
OK, assuming @kenoir that you've tried it and it runs successfully, I'm happy to merge.

**Comment by kenoir** [comment]
@philwills we're using this CF to run the app in the workflow AWS account. I'm not sure about backwards compatibility though.

**Comment by philwills** [comment]
Just tried applying this to the Ophan version which is in EC2 classic and got "Security groups can be applied to only ELBs in VPC.", which is a bit of a pain. I think for the moment we probably need to parameterise this, or have two templates, though neither of those options is terribly appealing.

We have been talking about moving everyone over to VPC as Amazon is definitely pushing us that way.

File: cloud-formation/status-app.json [hunk 1]
```diff
@@ -186,8 +209,12 @@
                 "SecurityGroupIngress" : [
                     { "IpProtocol": "tcp", "FromPort": "22",   "ToPort": "22",   "CidrIp": "0.0.0.0/0" },
                     { "IpProtocol": "tcp", "FromPort": "9000",   "ToPort": "9000",   "CidrIp": "77.91.248.0/21" },
-                    { "IpProtocol": "tcp", "FromPort": "9000",   "ToPort": "9000",
-                        "SourceSecurityGroupName": "amazon-elb-sg", "SourceSecurityGroupOwnerId": "amazon-elb" },
```
Comment (tackley) [inline cloud-formation/status-app.json]: yup. amazon-elb-sg is only for amazon classic, for amazon vpc you need to reference the security group that the load balancer is in:

```json
        "InstanceSecurityGroup" : {
            "Type" : "AWS::EC2::SecurityGroup",
            "Properties" : {
                "GroupDescription" : "Allow access from load balancer",
                "VpcId" : { "Ref" : "VPC" },
                "SecurityGroupIngress" : [
                    { "IpProtocol": "tcp", "FromPort": "9000", "ToPort": "9000", "SourceSecurityGroupId" : {"Ref": "ELBSecurityGroup"} },
                    { "IpProtocol": "tcp", "FromPort": "22",   "ToPort": "22",   "CidrIp": "...." }
                ]
            }
        },
```

**Comment by kenoir** [comment]
@philwills I went down the path of trying out conditions in the CF, work in progress: https://gist.github.com/kenoir/2a95ef5849bc6d012450

**Comment by kenoir** [comment]
@philwills after playing about I think trying to squeeze all the options (use vpc?/use ssl cert on load balancer?) is too complicated. I've created another CF config instead.

**Pull request merged**